setup.cfg

```diff
@@ -36,15 +36,16 @@ keywords =
 [options]
 zip_safe = True
 install_requires =
-    cobra
+    cobra==0.23
     depinfo
-    optlang<1.4.6
-    numpy
+    optlang>1.5
+    numpy==1.23
     scipy
     pandas
-    equilibrator-api==0.3.2b7
-    component-contribution==0.3.2b4
-    equilibrator-cache==0.3.2b2
+    equilibrator-api
```

Comment:
It would be nice to pin it, since the different equilibrator packages usually need to coordinated. This version set works for me with this branch:

```
equilibrator-api==0.6.0
equilibrator-cache==0.6.0
component-contribution==0.6.0
```

I think stating only `equilibrator-api>=0.6.0` and letting `pip` do the dependency resolution might also be fine.